sympy/holonomic/holonomic.py

```diff
@@ -1204,28 +1245,24 @@ def to_sequence(self, lb=True):
         >>> from sympy import symbols, S
         >>> x = symbols('x')
         >>> R, Dx = DifferentialOperators(QQ.old_poly_ring(x),'Dx')
-
-        # exp(x), the recurrence relation holds for n >= 0
         >>> HolonomicFunction(Dx - 1, x, 0, [1]).to_sequence()
         [(HolonomicSequence((-1) + (n + 1)Sn, n), u(0) = 1, 0)]
-
-        # log(1 + x), the recurrence relation holds for n >= 2
         >>> HolonomicFunction((1 + x)*Dx**2 + Dx, x, 0, [0, 1]).to_sequence()
         [(HolonomicSequence((n**2) + (n**2 + n)Sn, n), u(0) = 0, u(1) = 1, u(2) = -1/2, 2)]
-
         >>> HolonomicFunction(-S(1)/2 + x*Dx, x, 0, {S(1)/2: [1]}).to_sequence()
         [(HolonomicSequence((n), n), u(0) = 1, 1/2, 1)]
 
         See Also
         ========
 
-        HolonomicFunction.series
+        HolonomicFunction.series()
 
         References
         ==========
 
-        [1] hal.inria.fr/inria-00070025/document
-        [2] http://www.risc.jku.at/publications/download/risc_2244/DIPLFORM.pdf
+        .. [1]. https://hal.inria.fr/inria-00070025/document
```

Comment:
It seems that the full stop after `[1]` does confuse Sphinx. It does not recognize it.